examples/post_training/modelopt/README.md

```diff
@@ -102,6 +102,42 @@ deployment.
 
 See [Adanvanced Topics](./ADVANCED.md) for a `moonshotai/Kimi-K2-Instruct` EAGLE3 training example using `slurm`.
 
+### ⭐ Offline BF16 EAGLE3 Training
+Unlike online EAGLE3 training, offline workflow precomputes target model `hidden_states` and dumps to disk.
+Then only the draft model is called during training. AL is no longer reported during training. After training,
+`export.sh` is used to export EAGLE3 checkpoint.
+
+```sh
+\
+    HF_MODEL_CKPT=<pretrained_model_name_or_path> \
+    MLM_MODEL_SAVE=/tmp/Llama-3.2-1B-Eagle3 \
+    ./convert.sh meta-llama/Llama-3.2-1B-Instruct
+
+\
+    MLM_MODEL_CKPT=/tmp/Llama-3.2-1B-Eagle3 \
```

Comment:
can you add one comment before each command about what they do and will achieve?